src/script/descriptor.cpp

```diff
@@ -973,10 +973,10 @@ std::unique_ptr<PubkeyProvider> ParsePubkeyInner(uint32_t key_exp_index, const S
     }
     KeyPath path;
     DeriveType type = DeriveType::NO;
-    if (split.back() == MakeSpan("*").first(1)) {
+    if (split.back() == Span{"*"}.first(1)) {
```

Comment:
unrelated fun fact:

You could write `Span{std::string_view{"*"}}` and get rid of the `.first()` call.

(Same below)

Reply:
I don't think that's all that better.